modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -269,177 +383,212 @@ The output from this command lists the value for the key:
+image::os-tutorial-studio6-http-props.png[HTTP properties in Studio 6]
+. In the canvas, click *JSON to Object*, and then set *Return Class* to `java.util.Map`:
 +
-. Set the *Operation* to *Store* and set these *Store operation* values:
+.The arrow shows the *Return Class* field in the JSON to Object properties window.
+image::os-tutorial-studio6-json-props.png[JSON to Object properties in Studio 6]
+. In the canvas, click *ObjectStore*.
+. In the ObjectStore properties window:
+.. Set the *Operation* to *Store* and set these store operation values:
 +
-	* Set *Key* to `#[payload.key]`
-	* Set *Value Reference* to `#[payload.value]`
+* Set *Key* to `#[payload.key]`.
+* Set *Value Reference* to `#[payload.value]`.
+.. Click the green plus icon (+) to the right of *Connector Configuration*.
+.. In the ObjectStore Connector *Global Element Properties* window, set *Partition* to `myPartition` and *Object Store Reference* to `_defaultUserObjectStore`, and then click *OK*.
 +
-image::osv2-connector.png[]
+.The screenshot shows (*1*) the *Partition* field and (*2*) the *Object Store Reference* field in the *Global Element Properties* window.
+image::os-tutorial-studio6-os-global-props.png[Partion and Object Store Reference fields in the Global Element Properties window]
 +
-. Locate and add *Set Payload*. Set the *Value* field to `KEY: #[payload.key] AND VALUE: #[payload.value] WERE SAVED OK` - this message displays on the command line to let you know the curl command values are received by the application.
+[NOTE]
+The *Object Store Reference* field must be set to `_defaultUserObjectStore` for data to be stored in Object Store v2.
 +
-image::osv2-set-payload.png[]
+The ObjectStore properties window now indicates that there are no errors:
 +
-. Save the application and click *Run > Run As > Mule Application*. Running the application ensures your application works correctly. If you need to fix anything, do so now. After your application runs correctly, you can stop your application.
-
-=== To Deploy Your Application in Anypoint Platform
-
-. In Studio, right-click your application's name in Package Explorer and click *Anypoint Platform > Deploy to Cloud*.
-. In the user login window, specify your Anypoint Platform username and password, and click *Sign in*. If you don't have an Anypoint Platform login, click *Sign up*.
-. In Runtime Manager, specify an application name. Each name is unique and becomes the URL where your application is available in Runtime Manager. The URL has the format `APPLICATION_NAME.cloudhub.io` where your application resides in the cloudhub.io namespace. Ensure the application name has a green check mark for proper naming and being unique.
+.The arrow shows the ObjectStore properties window.
+image::os-tutorial-studio6-json-props.png[HTTP properties in Studio 6]
+. In the canvas, click *Set Payload*.
+. In the Set Payload properties windows, set the *Value* field to `KEY: #[payload.key] AND VALUE: #[payload.value] WERE SAVED`,
```

Comment:
Is Set Payload a label (*) and is there more than one window?

Reply:
period instead of comma

Reply:
Set Payload is an operation.